schemas/constructs/v1beta1/component.json

```diff
@@ -29,42 +29,113 @@
       "description": "Format specifies the format used in the `component.schema` field. JSON is the default."
     },
     "model": {
       "$ref": "./model.json",
       "description": "Reference to the specific registered model to which the component belongs and from which model version, category, and other properties may be referenced. Learn more at https://docs.meshery.io/concepts/models"
     },
-    "metadata": {
+    "styles": {
       "type": "object",
-      "description": "Metadata contains additional information associated with the component.",
-      "required": [
-        "shape",
-        "primaryColor",
-        "svgColor",
-        "svgWhite"
-      ],
+      "description": "Specify the styles",
+
+      "required": ["shape", "primaryColor", "svgColor", "svgWhite"],
       "properties": {
-        "capabilities": {
+        "primaryColor": {
+          "type": "string",
+          "description": "Primary color of the component used for UI representation."
+        },
+        "secondaryColor": {
+          "type": "string",
+          "description": "Secondary color of the component used for UI representation."
+        },
+        "svgWhite": {
+          "type": "string",
+          "description": "White SVG of the component used for UI representation on dark background."
+        },
+        "svgColor": {
+          "type": "string",
+          "description": "Colored SVG of the component used for UI representation on light background."
+        },
+        "svgComplete": {
+          "type": "string",
+          "description": "Complete SVG of the component used for UI representation, often inclusive of background."
+        },
+
+        "position": {
           "type": "object",
-          "description": "Meshery manages components in accordance with their specific capabilities. This field explicitly identifies those capabilities largely by what actions a given component supports; e.g. metric-scrape, sub-interface, and so on. This field is extensible. ComponentDefinitions may define a broad array of capabilities, which are in-turn dynamically interpretted by Meshery for full lifecycle management."
+          "description": "The position of the node. If the position is set, the node is drawn at that position in the given dimensions. If the position is not set, the node is drawn at a random position.",
+          "properties": {
+            "x": {
+              "type": "number",
+              "description": "The x-coordinate of the node."
+            },
+            "y": {
+              "type": "number",
+              "description": "The y-coordinate of the node."
+            }
+          }
         },
-        "genealogy": {
+
+        "body-text": {
           "type": "string",
-          "description": "Genealogy represents the various representational states of the component."
+          "description": "The text to display for an element\u2019s body. Can give a path, e.g. data(id) will label with the elements id"
```

Comment:
I wonder if use of unicode places any undue burden on clients that might display this string literally. Hopefully, most clients are interpreting and displaying the apostrophe.